### ISSUE TITLE
Hi there can i ask a question?

### ISSUE THREAD

**Issue by munkh-erdene** [NourozR/Stock-Price-Prediction-of-Apple-Inc.-Using-Recurrent-Neural-Network-#7]
First of all thank you for sharing your code. Im just looking for this type code (predict future price). Now im testing but i got wrong predicted values. All predicted values are increased. Why?

**Comment by kladskull** [comment]
He has no idea.

**Comment by KhaaQ** [comment]
Hi. Thank you for this project.. Although its  written in python 2.7. but there is a package named preprocessing and cannot be install in python2. what can be suitable alternate function instead of using preprocessing.new_data()? preferably in scikit learn.
Thank you

**Comment by mskhayati** [comment]
because he does not use Dropout function in keras.layers
from keras.layers import Dropout
after each layer LTSM (layer) use:
model.add(Dropout(0.2))

adjust the value with your results

**Comment by nubonics** [comment]
> First of all thank you for sharing your code. Im just looking for this type code (predict future price). Now im testing but i got wrong predicted values. All predicted values are increased. Why?

How did you predict more than one value at a time if you dont mind me asking?

**Comment by NourozR** [comment]
This was a very intro level code. If you keep changing dataset, it wouldn't work in the same way. Secondly, a lot of things has changed now in Keras and deep learning but basic things are still the same. Please check the recent documentation but like someone advised to use Dropout layers - please try that. 

Finally, someone said - "he has no idea" - to him, I would like to say - "Hey, I do". You can come and learn from me anytime. Thanks!
